components/skin/ui/src/main/resources/PhenoTips/DocumentMeta.xml

```diff
@@ -41,12 +41,12 @@
 #if ($doc.getObject('PhenoTips.PatientClass'))
   &lt;div id="document-authors"&gt;##
     &lt;span class="document-meta document-creation"&gt;## Creation
-      $services.localization.render('phenotips.patientRecord.header.creation', ["&lt;span class=""document-creator document-author""&gt;${xwiki.getUserName($tdoc.creator, false)}&lt;/span&gt;", $xwiki.formatDate($doc.creationDate)])##
+      $services.localization.render('phenotips.patientRecord.header.creation', ["&lt;a class=""document-creator document-author"" href=""${xwiki.getURL($tdoc.creator)}""&gt;${xwiki.getUserName($tdoc.creator, false)}&lt;/a&gt;", $xwiki.formatDate($doc.creationDate)])##
```

Comment:
If the creator is a guest (as is on the playground), then the link will point to the homepage. It should not be a link in that case.